cli/src/test/kotlin/de/deutschepost/sdm/cdlib/archive/ArchiveCommandIntegrationTest.kt

```
@@ -174,6 +174,7 @@ class ArchiveCommandIntegrationTest(@Value("\${artifactory-azure-identity-token}
         try {
             repository.delete(releaseName_build)
         } catch (_: Exception) {
+            // Exception is intentionally ignored here as it's a cleanup operation and non-critical.
```

The instruction was to add a comment in the empty catch block indicating that exceptions are intentionally ignored. This follows the given fix strategy to help other developers understand the reason behind ignoring exceptions, particularly in a cleanup operation where failure might be acceptable.<br><br>[Related Sonar Issue](http://localhost:9000/issues?open=15156638-4782-4aec-a774-3f852ff7c2ca)<br>[Related Sonar Rule](http://localhost:9000/coding_rules?open=kotlin:S108&rule_key=kotlin:S108)